mathext/digamma_test.go

```diff
@@ -29,7 +29,7 @@ func TestDigamma(t *testing.T) {
 		{-1.111111111e9, 30.497454343508262861},
 	} {
 
-		if got := Digamma(test.x); math.Abs(got-test.want) > 1e-10 {
+		if got := Digamma(test.x); math.Abs(got-test.want) > 1e-10*math.Abs(test.want) {
```

Comment:
We have `floats.EqualWithin*` for this kind of thing (they should have been used here in the original code). See [EqualWithinAbs](https://godoc.org/gonum.org/v1/gonum/floats#EqualWithinAbs), [EqualWithinRel](https://godoc.org/gonum.org/v1/gonum/floats#EqualWithinRel),  [EqualWithinRel](https://godoc.org/gonum.org/v1/gonum/floats#EqualWithinAbsOrRel),  and [EqualWithinULP](https://godoc.org/gonum.org/v1/gonum/floats#EqualWithinULP): take your pick.